resource-managers/nomad/src/main/scala/org/apache/spark/scheduler/cluster/nomad/ExecutorTask.scala

```diff
@@ -81,6 +83,14 @@ private[spark] object ExecutorTask
     // The driver will lookup the actual log URLs
     task.addEnv("SPARK_LOG_URL_" + LOG_KEY_FOR_ALLOC_ID, "${NOMAD_ALLOC_ID}")
 
+    if (conf.getOption("spark.jars").nonEmpty) {
+      task.addEnv("SPARK_EXECUTOR_CLASSPATH", conf.getOption("spark.jars").get
+        .split(",").map(jarLocation => new URI(jarLocation).getPath).mkString(":"))
+    }
+    if (conf.getOption("spark.executor.extraClassPath").nonEmpty) {
+      task.addEnv("SPARK_EXECUTOR_CLASSPATH", conf.getOption("spark.executor.extraClassPath").get)
+    }
+
```

Comment:
Small question here - if a user were to put both `spark.jars` and `spark.executor.extraClassPath` options, only the latter would exist within the task env variable `SPARK_EXECUTOR_CLASSPATH`. Are we OK with that, or should the `spark.executor.extraClassPath` be appended to the existing content of the classpath?

Reply:
yes, `spark.executor.extraClassPath` values override values of `spark.jars`

I was following `Mesos` Backend.
https://github.com/ExpediaGroup/nomad-spark/blob/nomad-spark-2.4.3/resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackend.scala#L226-L229

I initially did not had the code for `spark.jars`, but later added to handle the case - If User passed only one property `spark.jars`.

Reply:
I don't think the values provided in `spark.jars` are ever intended to be added to any classpaths at runtime, they are downloaded after the driver/executor is already running and classloaded

Reply:
Removed handling the spark property `spark.jars`.  thanks.
also verified, `MesosCoarseGrainedSchedulerBackend` too does not do anything specific to property `spark.jars`